src/components/AppMainView.tsx

```diff
@@ -46,25 +44,23 @@ const AppMainView = () => {
             <WarningView />
             <div className="memory-box-container">
                 <MemoryBoxView
-                    title="File memory layout"
-                    description="Drag & drop HEX/ZIP files here"
-                    iconName="mdi mdi-folder-open"
+                    title="Memory visualization"
+                    description="Drag & drop HEX/ZIP/ELF files here"
+                    iconName="mdi mdi-file-outline"
```

Comment:
What is the reason for changing the title and icon?

Reply:
I would think it's more intuitive that you're looking at an elf-file if there's a file icon + extension, than if you just see a directory/folder icon. But the title change should be reverted, thanks for catching that.

Reply:
As it is right now it changes the default behaviour so that mdi-file-outline is shown also for all other states (like no files or other file types).
Do ask Ketil though, maybe he thinks this is how it should be.

Reply:
![image](https://user-images.githubusercontent.com/55808963/187426970-7a2a6b56-96e9-4a12-9c43-e0bbe53babea.png)
Ketil approves

Reply:
1337

Reply:
I will work on this so it works with hex and zip files too